aip/classic/read_records.py

```diff
@@ -3,7 +3,7 @@
 import time
 import timeout_decorator
 
-from aip.classic import xmltodict
+import xmltodict
```

Comment:
oho! very nice catch, not very nice of the previous dev to not include a comment with a url